doc/unbound.conf.5.in

```diff
@@ -2114,6 +2114,78 @@ If this timeout expires Unbound closes the connection, treats it as
 if the Redis server does not have the requested data, and will try to
 re-establish a new connection later.
 This option defaults to 100 milliseconds.
+.SS DNSTAP Logging Options
+DNSTAP support, when compiled in, is enabled in the \fBdnstap:\fR section.
```

Comment:
Maybe add that dnstap will create a thread, as this might change the number you want to configure for num-threads.

(I realize this was also the case in the old implementation)

Reply:
Yes good to note. The old implementation did not have documentation, so I added it. Fixed in https://github.com/NLnetLabs/unbound/pull/164/commits/e24d7c64a8e13160d3fee2c6fe9f9cbf952fcc3b .